js/components/channel_messages.jsx

```diff
@@ -23,11 +23,21 @@ var ChannelMessages = React.createClass({
   },
 
   componentDidMount: function() {
-    MessageStore.on('change', this.messageStoreChange);
+    MessageStore.on('change', this.handleMessageChange);
   },
 
   messageStoreChange: function() {
     this.setState({messages: MessageStore.all()});
+  },
+
+  handleMessageChange: function(){
```

Comment:
"handleMessageChange" is a little confusing since individual messages aren't what's changing. Maybe make it messageStoreChange or handleMessageStoreChange?